android/src/main/java/com/onexip/plugins/capacitorscreensize/ScreenSizePlugin.java

```diff
@@ -16,14 +16,14 @@
     public void getDevicePPI(PluginCall call) {
       DisplayMetrics displayMetrics = this.getActivity().getResources().getDisplayMetrics();
 
-      String xdpi = Float.toString(displayMetrics.xdpi);
+      String density = Float.toString(displayMetrics.density);
       String ydpi = Float.toString(displayMetrics.ydpi);
 
       JSObject ret = new JSObject();
-      ret.put("xdpi", xdpi);
-      ret.put("ydpi", ydpi);
+      ret.put("density", ydpi);
+      ret.put("scaleFactor", density);
 
-      //System.out.println("#### ret: " + ret.getString("xdpi") + ", " + ret.getString("ydpi"));
+      //System.out.println("#### density: " + ret.getString("density") + ", scaleFactor: " + ret.getString("scaleFactor"));
```

Comment:
we should not let commented out code here

Reply:
removed commented code